dir.props

```diff
@@ -80,6 +81,17 @@
     <ProdConManifestFile>$(PackageReportDir)prodcon-build.xml</ProdConManifestFile>
     <PoisonedReportFile>$(PackageReportDir)poisoned.txt</PoisonedReportFile>
     <ConflictingPackageReportDir>$(BaseOutputPath)conflict-report/</ConflictingPackageReportDir>
+    <ReferencePackagesBasePath>$(IntermediatePath)</ReferencePackagesBasePath>
+    <!--
+      Change ReferencePackagesBasePath conditionally in offline build.
+      See corresponding change in build-source-tarball.sh to copy reference-packages
+      from source-build bin dir to tarball reference-packages dir.
+    -->
+    <ReferencePackagesBasePath Condition="'$(OfflineBuild)' == 'true'">$(ProjectDir)</ReferencePackagesBasePath>
```

Comment:
I think this would be clearer and remove some duplication:

```xml
    <ReferencePackagesBasePath>$(IntermediatePath)reference-packages/</ReferencePackagesBasePath>
    <!--
      Change ReferencePackagesBasePath conditionally in offline build.
      See corresponding change in build-source-tarball.sh to copy reference-packages
      from source-build bin dir to tarball reference-packages dir.
    -->
    <ReferencePackagesBasePath Condition="'$(OfflineBuild)' == 'true'">$(ProjectDir)reference-packages/</ReferencePackagesBasePath>
    <ReferencePackagesStagingDir>$(ReferencePackagesBasePath)staging/</ReferencePackagesStagingDir>
```

Technically duplicating the dir name, but I think it's fine and makes it clear what the prop's pointing to in context.

(I'd also call it `ReferencePackagesBaseDir` rather than `Path`. 😄)

Reply:
Agreed. Thanks.